docs/backplane/guideline.md

```diff
@@ -93,6 +93,7 @@ Resource specific guidelines below are enforced by CI where possible.
 The following secrets may be accessed by the respective persona for the given reasons.
 Note to be in this list the `Secret` must not contain any secret data (see [Definitions](#definitions)).
 
-| Persona | Namespace | Secret | Reason |
-| --- | --- | --- | --- |
-| SREP | openshift-monitoring | alertmanager-main | Platform SRE have access to the PagerDuty API and DMS Webhook from other systems. The ability to review the configuration of alertmanager is important for SRE to understand how monitoring has been configured and if there are issues with the routing of alerts. |
+| Persona        | Namespace            | Secret                    | Reason                                                                                                                                                                                                                                                              |
+|----------------|----------------------|---------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SREP           | openshift-monitoring | alertmanager-main         | Platform SRE have access to the PagerDuty API and DMS Webhook from other systems. The ability to review the configuration of alertmanager is important for SRE to understand how monitoring has been configured and if there are issues with the routing of alerts. |
+| SRE or ACS eng | rhacs-observability  | grafana-admin-credentials | SREs and ACS engineers need admin access to Grafana to tweak dashboards. Changes to dashboards are temporary, as they are ultimately reconciled by [RHACS Observability Resources Repository](https://github.com/stackrox/rhacs-observability-resources).           |
```

Comment:
The requirements and role configuration do not align with this statement which implies a need for `edit` permissions.  Is there some other documentation or a user story that references the use case for viewing and/or modifying this secret?  Is it done as a normal thing?  What secret data is exposed in that secret when someone does view it?  How quickly does the reconcile happen (if edit is really being requested)?

Reply:
We only need access to read this secret. We manage our dashboards through gitops (within this repository). But, it's next-to-impossible to edit the grafana JSON dashboards manually (it's extremely verbose). So we usually login as admin to grafana (hence the need to read this secret), modify the dashboards, export them as json, and issue a MR to the repository.

Reply:
When we merge changes to the repository, it gets automatically into the `integration` cluster. We have a GHA to deploy the changes to the 2x`stage` and 2x`prod` clusters.

We don't really have a formalized document on how to do this AFAIK, it's something we usually do ad-hoc. fyi: @stehessel

Reply:
> What secret data is exposed in that secret when someone does view it?
It's the password for logging into Grafana as `admin`. But our grafana instance is protected by SSO, so the password alone is not sufficient to login.

Reply:
The branches and promotion setup are described in the README in https://github.com/stackrox/rhacs-observability-resources. The Grafana admin password is an unfortunate result of the operator setup we are using. Ideally we'd like to remove this password and just give any successfully authenticated user admin rights on the dashboards. We have plans to simplify the login setup this way, then the secret will no longer exist.